src/nvim/tag.c

```diff
@@ -1947,6 +1947,7 @@ static void findtags_add_match(findtags_state_T *st, tagptrs_T *tagpp, findtags_
   const bool name_only = (st->flags & TAG_NAMES);
   int mtt;
   size_t len = 0;
+  size_t allocation_size = 0;
```

Comment:
`mfp_size` would be a more conventional name for this, and gives a hint about its purpose.

Reply:
> `mfp_size` would be a more conventional name for this, and gives a hint about its purpose.
You're right. I've changed it.